app.js

```diff
@@ -0,0 +1,13 @@
+"use strict";
+
+// simple express server
+var express = require("express");
+var app = express();
+var router = express.Router();
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [A function with a name starting with an uppercase letter should only be used as a constructor.](https://www.codacy.com/app/lagora-franck/proc-edu-ria/file/2546558170/issues/source?bid=3286583&fileBranchId=3287652#l6)